trellis-webac/build.gradle

```diff
@@ -9,6 +9,7 @@ ext {
 dependencies {
 
     api group: 'org.apache.commons', name: 'commons-rdf-api', version: commonsRdfVersion
+    api group: 'org.apache.servicemix.bundles', name: 'org.apache.servicemix.bundles.javax-inject', version: javaxInjectVersion
```

Comment:
Is it this (and not the vanilla API JAR) in order to get OSGi metadata?

Reply:
Yes, it's just to make it easier to deploy in Karaf.